src/App.js

```diff
@@ -53,40 +59,66 @@ function App() {
       })
     });
     const data = await response.json();
-    console.log(data);
     //split the json reponse by dashes into an array 
-    const pointsArray = data.data.split("-");
+    const pointsArray = data.data;
     setSummaryPoints(pointsArray);
     setScript("");
+    handleCreate(title);
   }
 
+  async function handleCreate(title){
+    //makes http request to create presentation
+    try {
+      const response = await fetch('http://localhost:5000/create-presentation', {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({ title }),
+      });
+      const data = await response.json();
+      return data;
+    } catch (error) {
+      console.error(error);
+    }
+  }
+
+
 
   return (
     <div>
       <h1>Google Slides Automator</h1>
       <FileUpload onFileUploaded={handleFileUploaded} />
-      {/* <GoogleSlidesPresenter summaryPoints={summaryPoints} />  */}
-      <br/>
+      {/*<GoogleSlidesPres />  */}
+      <br />
       <strong>OR</strong>
-      <p>Paste the script here</p>
+      <p>Paste the script below</p>
       <form onSubmit={handleSubmit}>
-      <textarea 
-      type="text"
-      placeholder="Enter Script"
-      cols={80}
-      rows={10}
-      onChange={handleChange} 
-      onPaste={handlePaste}
-      value={input}/><br />
-      {input !== "" ? (<button type="submit">Submit</button>) : null}
+        <strong>Title: </strong>
+        <input
+        type="text"
+        placeholder="Enter Title"
+        onChange={handleChangeTitle}
+        value={title}>
+        </input><br /><br/>
+        <strong>Script: </strong>
+        <textarea
+          type="text"
+          placeholder="Enter Script"
+          cols={80}
+          rows={10}
+          onChange={handleChange}
+          onPaste={handlePaste}
+          value={input} /><br />
+        {input !== "" ? (<button type="submit">Submit</button>) : null}
```

Comment:
instead of conditionally rendering this button, we can use the `disabled` prop

```suggestion
        <button type="submit" disabled={input !== ""}>Submit</button>
```